README.md

```diff
@@ -31,6 +31,7 @@ Job specifications are passed through env variables:
   - [Required] _auth_token_: User token authentication.
   - [Required] _api_host_: API host URL.
   - [Optional] _args_: Dictionary with job arguments.
+  - [Required] _collection_: String with name of collection where items will be stored.
```

Comment:
Just for the good format, I think this line should go above args, to have optional at the end.

Reply:
Please don't forget about this 😅